proto/cosmos/crypto/secp256k1/keys.proto

```diff
@@ -6,17 +6,37 @@ import "gogoproto/gogo.proto";
 
 option go_package = "github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1";
 
-// PubKey defines a secp256k1 public key
+// PubKeyOld defines a tendermint secp256k1 public key
 // Key is the compressed form of the pubkey. The first byte depends is a 0x02 byte
 // if the y-coordinate is the lexicographically largest of the two associated with
 // the x-coordinate. Otherwise the first byte is a 0x03.
 // This prefix is followed with the x-coordinate.
```

Comment:
Let's also replace the comment with the one for uncompressed public key.

Reply:
What do you mean?
The other one's comment is already updayed
```
// PubKey defines a comet uncompressed secp256k1 public key
// Key is the uncompressed form of the pubkey.
message PubKey {
```

Reply:
I let this one be `tendermint secp256k1 public key` as the amino name is still tendermint related `(amino.name) = "comet/PubKeySecp256k1Uncompressed";`

Reply:
The `PubKeyOld` is also an uncompressed public key. Maybe we can say both `PubKeyOld` and `PubKey` types are inherently the same.

Reply:
Yeah let's use the same comment. Addressed [here](fdf810ea8b6823862e3126b01313f3707dcda6e2)